### PR TITLE
LRAC-14487 | master

### PR DESCRIPTION
**Pull request by lesliewong92**
https://liferay.atlassian.net/browse/LRAC-14487

### CODE REVIEW

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/model/preferences/WorkspacePreferences.java [hunk 1]
```diff
@@ -41,6 +41,14 @@ public Map<String, EmailReportPreferences> addEmailReportPreference(
 		return _emailReportPreferences;
 	}
 
+	public Map<String, EmailReportPreferences> deleteEmailReportPreference(
```
Comment (rachaelkoestartyo) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/model/preferences/WorkspacePreferences.java:44]: let's make it plural `deleteEmailReportPreferences`

Comment (lesliewong92) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/model/preferences/WorkspacePreferences.java]: Renamed to `removeEmailReportPreference` to match `removeIndividualSegmentPreference`

**Comment by marcosapmf** [comment]
ci:test:sf

**Comment by marcosapmf** [comment]
ci:test:relevant

**Pull request closed without merging**